client/src/style/main.pcss

```diff
@@ -10,6 +10,14 @@
     line-height: 24px;
   }
 
+  /* this was SO ANNOYING. Something in base Tailwind css sets bg-color to transparent and I had so much frustrating overriding it. hence this...but I want a more appropriate solution */
+  button,
```

Comment:
I think this is pretty much the right approach -- I'm checking with Kerrin, I think what you stumbled across may be a bug in the tailwindcss-forms package